### PR TITLE
Update user guide after PED

### LINKED ISSUE #127: [PE-D] School tag could be in lower caps or stated somewhere that lower cases are allowed
I understand that since these are short forms for NUS schools, they are represented in capitals such as `SCIENCE` and `BIZ` but,

Since the user can add / edit schools with inputs in lower case as well `s/biz` or `s/soc`, it might be good to mention that somewhere clearly for the user either in the UG or Help Window.

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: VRSoorya/ped#9

### LINKED ISSUE #128: [PE-D] Better explanations for expected behaviour of addon command
This was the only bullet under `addon` that hinted that there could be price or price range to a entry:

However, I could not tell that `addon 1 p/5` when entry 1 had price 4 would give a range instead such as `$4-5` after the addon. Since there is also an `edit` command that can edit price attributes of entries, I think it'll be clearer by adding specific examples to show before and after values in these 2 commands for users to understand the difference in these commands quickly.

Marking severity as 'low' since there might be unintended inconvenience to the user (especially still a new user on 2nd or 3rd use) if they mixed up the use of `edit` and `addon` as users would be naturally inclined to think of `edit` first when wanting to update an entry.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: VRSoorya/ped#10

### LINKED ISSUE #129: [PE-D] UG does not list down the prefix and their usage.
It would be better to give a list of prefix for the fields. 
This can make the reader of the UG clearer of what prefix to use when they are typing into the command line. Otherwise, they have to look at the examples in order to figure out what prefix to use.

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: icytornado/ped#1

### LINKED ISSUE #131: [PE-D] Delete command bug: not allowing deletion by name
List of commands to reproduce:
1. clear 
2. add  n/7 Eleven ra/3 p/4 re/Mac and cheese there is amazing a/NUS c/Halal s/YIH s/SOC
3. delete n/7 Eleven

Expected: the entry containing 7 Eleven as restaurant is deleted.

Actual: the entry is not deleted, and an invalid command warning is shown. The entry should be deleted, as stated in the UG, because the UG says can delete a restaurant by it's name.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: icytornado/ped#3

### LINKED ISSUE #132: [PE-D] Add command bug: allowing addition of restaurants with different names, but exactly same location
List of commands to reproduce:
1. clear
2. add  n/Al Amaan Restaurant ra/5 p/8 re/best for Butter Chicken a/12 Clementi Rd, Singapore 129742 c/Indian s/FOE
3. add  n/KFC ra/5 p/8 re/best for Butter Chicken a/12 Clementi Rd, Singapore 129742 c/Indian s/FOE

Expected: since the second entry has the same address as the first one. It should not be allowed to be added in, because otherwise, there will be location clash.

Actual: the second entry is allowed to be added in.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: icytornado/ped#4

### LINKED ISSUE #133: [PE-D] Add command bug: not allowing addition of restaurants with same name but different location
List of commands to reproduce:
1. clear
2. add n/Subway ra/5 p/6 re/I like this food a lot! a/3155 Commonwealth Ave W, Singapore 129588 c/FastFood c/Vegan s/SOC
3.add n/Subway ra/5 p/6 re/I like this food a lot! a/NUS c/FastFood c/Vegan s/FOE

Expected: the second entry should be allowed to be added in. Although the two restaurants have the same name, they have different location. This should be fine, because they are just two Subway restaurants at two locations.

Actual: An "This entry already exists in the food diary" is shown.

comments: the makes the product almost unusable, because currently the users are only able to  add in a restaurant name that is different than the existing ones.

-------------
Labels: `severity.High` `type.FunctionalityBug`
original: icytornado/ped#5

### LINKED ISSUE #138: [PE-D] Not enough example given for delete command
Not enough example. The example under delete, its better to include at least one example for every type of commands, so that the user has a more straightforward idea on what to type. 

should add in the command example: delete 1. currently, only the example for delete restaurant name is shown, the example for delete by index is not shown.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: icytornado/ped#10

### LINKED ISSUE #142: [PE-D] User guide inconsistent description bug
Under the description for delete command: "Delete i/ INDEX" should be "delete INDEX" instead, because when the user trys to delete something, they just type in delete INDEX, without the prefix. For example, delete 1 . The one shown in the command summary is fine, which is delete INDEX.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: icytornado/ped#14

### LINKED ISSUE #147: [PE-D] Unclear what the angle brackets and square brackets mean
In the user guide, square brackets and angle brackets(and also sometimes no brackets) were being used in the format section of the commands. But it was not explained anywhere in the user guide what these brackets mean.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: IceBear789/ped#2

### LINKED ISSUE #148: [PE-D] The list command section is duplicated
In the user guide, the usage for the list command is displayed twice even though there are no notable differences between the 2 sections.

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: IceBear789/ped#3

### PR DESCRIPTION
**Pull request by Sidney011100**

### CODE REVIEW

**Comment by Sidney011100** [comment]
closed issues with pull request that could not be linked #125, #123, #122, #121

File: docs/UserGuide.md [hunk 1]
```diff
@@ -30,18 +30,30 @@ The Food Diary will allow students to save time and effort when finding places t
...
+* Parameters can be in any order.
+e.g. if the command specifies n/NAME p/PRICE, p/PRICE n/NAME is also acceptable.
+
+* If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.
+e.g. if you specify p/5 p/6, only p/6 will be taken.
+
+* Extraneous parameters for commands that do not take in parameters (such as help, list, exit and clear) will be ignored.
+e.g. if the command specifies help 123, it will be interpreted as help.
+  
+* [c/CATEGORY] needs to be titled case, [s/SCHOOL] can be lower or upper case
```
Comment (Donavanty) [inline docs/UserGuide.md:55]: Good write-up, will check again to see if it mentions everything that's needed to be mentioned. Thank Sidney

File: docs/UserGuide.md [hunk 2]
```diff
@@ -157,18 +170,18 @@ Finds for FoodDiary entries whose names, ratings, price, address, categories and
...
-6. `Categories`
-7. `Schools`
+PARAMETER(s) of any number and sequence:
+1. To search by food place name: `<Name>`
+2. To search by rating: `<Rating>/5`
+3. To search by price: `$<Price>`
+4. To search by price range: `$<Price-Price>`
+5. To search by address: `<Address>`
+6. To search by categories: `<Categories>`
+7. To search by schools: `<Schools>`
```
Comment (Donavanty) [inline docs/UserGuide.md:184]: Good. Should be clearer

**Review by Donavanty — APPROVED** [review]
LGTM.

**Pull request merged**